### DISCUSSION TITLE
Comment in yuck?

### DISCUSSION THREAD

**Discussion by winston0410** [elkowar/eww#282] [Q&A]
I find the comment token of yuck here, but what is it exact syntax?

https://github.com/elkowar/eww/blob/9c12a316d6e3c3b1b504bf1ed86de5241694ef2a/crates/yuck/src/parser/parser.lalrpop#L25

**Comment by VuiMuich** [comment]
```clojure
; line comment
#| Multi
Line
Comment |#
```
Iirc

**Reply by winston0410** [reply]
Perfect. I will add support for this in my own comment plugin in neovim. Btw I guess the grammar of yuck Is largely similar with Lisp, except that multiline comment? I think we can easily create a tree-sitter grammar for it?

**Reply by VuiMuich** [reply]
At some point there was a recommendation for parinfer and a syntax-highlighting plugin for vim/neovim, but I can't find the links right now.
But yes, yuck being a simple expression language has lots of similarities with lisp and clojure.

Edit: found it, quoting from [configuration documentation](https://elkowar.github.io/eww/configuration.html)
> Yuck is based around S-expressions, which you may know from lisp-like languages.
If you're using vim, you can make use of [yuck.vim](https://github.com/elkowar/yuck.vim) for editor support.
It is also recommended to use [parinfer](https://shaunlebron.github.io/parinfer/),
which makes working with s-expressions delightfully easy!

**Reply by winston0410** [reply]
Just did a creepy search, seems like no repo named `tree-sitter-yuck` right now, but I find this dotfile is using its own version of `tree-sitter-yuck` in local. Let me ask him if he want to push this so that we can all use and contribute to this repo.

https://github.com/search?q=tree-sitter-yuck&type=code
